tools/VscPreviewExtension/src/dfmService.ts

```diff
@@ -3,28 +3,50 @@
 import { AxiosError } from 'axios';
 
 import { DfmHttpClient } from './dfmHttpClient';
-import { DfmServiceResult } from './dfmServiceResult';
+import * as ConstVariable from "./ConstVariable";
 
 export class DfmService {
-    private static client = new DfmHttpClient();
-
-    static async previewAsync(content: String): Promise<DfmServiceResult> {
+    static preview(docfxServicePort, content: String, workspacePath, relativePath, writeTempPreviewFile = false, previewFilePath = null, pageRefreshJsFilePath = null, builtHtmlPath = null) {
         if (!content) {
             return null;
         }
 
-        return await DfmService.client.sendPostRequestAsync("preview", content);
+        return new Promise(function (fulfill, reject) {
```

Comment:
Nee some investigation.

---
In reply to: [111536265](https://github.com/dotnet/docfx/pull/1547#discussion_r111536265) [](ancestors = 111536265)